infinex-patron/patron-purchase-app/mainnets/infinex-patron.arbitrum.toml

```diff
@@ -0,0 +1,68 @@
+name = "infinex-patron"
+preset = "patron-purchase"
+version = "1.0.0"
+description = "Smart Contracts used for Infinex Patron Purchase App"
+
+# pnpm cannon build infinex-patron/patron-purchase-app/mainnets/infinex-patron.arbitrum.toml --chain-id 42161 --dry-run
+
+[var.PATRON]
+depends = []
+PATRON_DEPLOYER = "0xCD8AeD7674B10e1b6c8281B91259CEDE1BFE4a19"
+AUTHORIZED_SIGNER = "0x4f61e5795709C7A7E950Ea48c1cF5035F596d8A0"
+# AUTHORIZED MULTISIGS
+MULTISIG_0 = "0x4B190EC7A62EcE54c4DbF24F85D4fceF66c1446B"
+# AUTHORIZED TOKENS
+USDC_TOKEN = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831" 
+USDT_TOKEN = "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9"  
+USDE_TOKEN = "0x5d3a1Ff2b6BAb83b63cd9AD0787074081a52ef34"
+USDC_E_TOKEN = "0xff970a61a04b1ca14834a43f5de4533ebddb5cc8"
+DAI_TOKEN = "0xda10009cbd5d07dd0cecc66161fc93d7c9000da1"
+ARB_TOKEN = "0x912CE59144191C1204E64559FE8253a0e49E6548"
+ETHER = "0x0000000000000000000000000000000000000000"
+
+[clone.infinexPatronPurchase]
+depends = ['var.PATRON']
+options.PATRON_DEPLOYER = "<%= settings.PATRON_DEPLOYER %>"
+options.PATRON_PURCHASE_SALT = "PATRON_PURCHASE_SALT_1_0_13"
+source = "infinex:1.0.1@patron-purchase"
+target = "infinex:1.0.1@patron-purchase"
+
+[invoke.PatronPointOfPurchaseInitialize]
+target = ["infinexPatronPurchase.PatronPointOfPurchase"]
+func = "initialize"
+args = [
+    "<%= settings.PATRON_DEPLOYER %>",
+    "<%= settings.MULTISIG_0 %>", # if funds get stuck, this is where they go.
```

Comment:
good practice for this is creating another set of settings where it's the arguments that are being passed to the contracts, and making that setting the multisig. e.g.
```
args = [
    "<%= settings.PATRON_OWNER %>",
    "<%= settings.PATRON_FUNDS_DESTINATION %>", # if funds get stuck, this is where they go.
```
```
[var.PatronSettings]
PATRON_OWNER = "<%= settings.PATRON_DEPLOYER %>"
PATRON_FUNDS_DESTINATION = "<%= settings.MULTISIG_0 %>"
``` 

this isn't strictly necessary but as you can see, it greatly improves the readability

Reply:
Agreed on this, I heard this might not be possible due to circular references?